lobby.js

```diff
@@ -0,0 +1,193 @@
+var LobbyFactory = require('./app/factories/LobbyFactory.js');
+"use strict";
+var port = 1234;
+const io = require('socket.io')(port);
+const repl = require('repl');
+
+const name = "My Lobby";
+const creator = "molen";
+const gamemode = "URF";
+
+const adminSettings = [{
+    binding: "available-champions",
+    name: "Availble Champions",
+    help: "Champions available",
+    field: "championSelectMulti"
+}, {
+    binding: "available-spells",
+    name: "Available Spells",
+    help: "Spells available",
+    field: "summonerSpellSelectMulti"
+}, {
+    binding: "map",
+    name: "Map",
+    help: "Foo",
+    field: "mapSelect",
+    default: 11,
+    options: "*"
+}];
+
+const playerSettings = [{
```

Comment:
@MythicManiac I dont understand this so I wont edit nothing until I understand all. Also this is not revelant